spec/config/sgtnclient.yml

```diff
@@ -24,6 +24,9 @@ test: &default
   # # the lowest level to be logged, default value is INFO
   log_level: DEBUG
 
+  # # tag for pseudo translation, default value is '@@'
+  pseudo_tag: "@@"
```

Comment:
no double quotes?

Reply:
Looks @@ has special meaning in YAML, the double quotes are necessary.